client/src/components/Navbar/index.jsx

```diff
@@ -6,7 +6,9 @@ const Navbar = () => {
   return (
     <nav className="primary-nav">
       <div>
-        <img src={Logo} alt="Roadmap Runner logo" />
+        <a href="/">
+          <img src={Logo} alt="Roadmap Runner logo" />
+        </a>
```

Comment:
Can you use the `Link` component of the `react-router-dom` instead of the anchor tag?